ci-operator/config/konflux-workspaces/workspaces/konflux-workspaces-workspaces-main.yaml

```diff
@@ -0,0 +1,39 @@
+build_root:
+  project_image:
+    dockerfile_path: ci/Dockerfile
+releases:
+  initial:
+    integration:
+      name: "4.15"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.15"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 250Mi
+tests:
+- as: e2e
+  run_if_changed: ^.github/workflows/push.yaml$|^server|^operator|^ci
+  steps:
+    cluster_profile: aws-rhtap-qe
+    test:
+    - as: e2e
```

Comment:
Similar to my first comment. As of right now this works for a single test/job that you want to run but if you plan to leverage what this step is doing in any other config you'll need to copy and paste this code in each new config. Example: Some other team might want to run this test as a part of an integration test between a stack of many deployed products.

We can use the step-registry to hold repeatable tests that can be easily leveraged. This allows others to use our tests if ever needed and prevents us from ever needing to copy/paste test steps in our own configs.

Sticking with the same example in the first comment which promoted the clc-ui-e2e image, we see that image being used in [this step in the step-registry](https://github.com/openshift/release/tree/93352c979a01cb17dcdcc30dd1dfb7463e56325e/ci-operator/step-registry/acm/tests/clc-create).

2 files are needed
- .yaml file to tell this step instructions about the execution environment (image to use, variables, secrets to mount, ..etc)
- commands.sh script to hold the execution steps. lines 28-30 that  you have below would go here.

Then you can just call the step/ref that you created in your config file [like this](https://github.com/openshift/release/blob/93352c979a01cb17dcdcc30dd1dfb7463e56325e/ci-operator/config/stolostron/acmqe-autotest/stolostron-acmqe-autotest-main__acm-ocp4.16-lp-interop.yaml#L87)

Reply:
I'm of a similar mindset here; I think this would be nice to have, but I'm thinking I'll punt this to a future PR if you're okay with that.  As much as I want to make this change, I think I'd like to get end-to-end tests working in CI first.

Reply:
Again I agree, it can be nice to get things running first then decide how to optimize if the need arises later.